web/packages/shared/components/UnifiedResources/shared/SingleLineBox.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+
+import Box from 'design/Box';
+
+export const SingleLineBox = styled(Box)`
```

Comment:
This has been taken out to a separate file but so far it's used only in card view, I'd consider moving it back unless it's needed for an upcoming PR.

Reply:
yeah it's being used in an upcoming PR